src/alfasim_sdk/__init__.py

```diff
@@ -462,6 +463,7 @@ def get_alfasim_sdk_api_path():
     "PvtModelCompositionalDescription",
     "PvtModelCorrelationDescription",
     "PvtModelTableParametersDescription",
+    "PhPvtModelTableParametersDescription",
```

Comment:
I thought that this would be automatically formatted to be added in alphabetical order. It seems that we don't do that. Therefore, could you move this to before `"PhysicsDescription"`, these in theory should be in alphabetical order.